packages/compass-indexes/src/components/search-indexes-table/search-index-actions.tsx

```diff
@@ -45,11 +59,21 @@ const IndexActions: React.FunctionComponent<IndexActionsProps> = ({
   );
 
   return (
-    <ItemActionGroup<SearchIndexAction>
-      data-testid="search-index-actions"
-      actions={indexActions}
-      onAction={onAction}
-    ></ItemActionGroup>
+    <div className={actionGroupStyles}>
+      <Button
+        data-testid="search-index-actions-aggregate-action"
+        className={cx([!index.queryable && css({ visibility: 'hidden' })])}
```

Comment:
Please don't add css calls in component runtime, this can be defined outside of the component. Also `cx` function takes a spread of arguments, you don't necessarily need to pass array here, it can be just one argument

Reply:
oh yeah! somehow missed it.

Reply:
fixed in [004be9c](https://github.com/mongodb-js/compass/pull/4893/commits/004be9c1a673c33a18773cb6dee681f72795ac2e)